app/helpers/shipping_methods_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ShippingMethodsHelper
+  def shipping_methods
+    ShippingMethod.order('price ASC')
```

Comment:
@b-artem it's vey bad idea to use activerecord in helpers. If you need data in template - load it in controller

Reply:
Done

Reply:
Done